sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

```diff
@@ -82,6 +82,98 @@ With this configuration, the Azure Monitor Distro will use the credentials of th
 
 Note that the `Credential` property is optional. If it is not set, Azure Monitor Distro will use the Instrumentation Key from the Connection String to send data to Azure Monitor.
 
+### Advanced configuration
+
+The Azure Monitor Distro includes .NET OpenTelemetry instrumentation for [ASP.NET Core](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.AspNetCore/), [HttpClient](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.Http/), and [SQLClient](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.SqlClient). However, you can customize the instrumentation included or use additional instrumentation on your own using the OpenTelemetry API. Here are some examples of how to customize the instrumentation:
+
+#### Customizing ASP.NET Core Instrumentation Options
```

Comment:
consider linking to the instrumentation library readme here to help users find the full options available

Reply:
We need to link this to a GitHub page, which might change. Better we get into this details when we publish docs in docs.microsoft.com.